MobileConnectionLibrary/MobileConnectionLibrary/Subscriber.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MobileConnectionLibrary
+{
+    public class Subscriber
+    {
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public long PhoneNumber { get; set; } // длинный инт
+
+        public readonly int ContractNumber;
+
+        private string tariff_name; //поле
+        public string TariffName //свойство
+        {
+            get
+            {
+                return tariff_name;
+            }
+
+            set
+            {
+                tariff_name = value;
+            }
+        }
```

Comment:
Если не нужен контроль целостности данных, можно использовать упрощенный синтаксис, как вы его использовали выше
```suggestion
        public string TariffName { get; set; }
```